### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by nowarkdev**
A few bits and pieces on this PR:

1. Fix font.css file - needs to be .scss file now as its no longer plain css. Removed indentedSyntax in webpack.config.js as it was forcing sass complication instead of scss

2. A little tidy up with indentation on docs index - there was no </section>, </body>, </html> at the end of the file plus some tags were not properly indented. 

3. Add typography section to styleguide - We have used the latest UI designs to correctly set size, font weight, etc. Having spoken to design regarding vertical margins we have been told that these are contextual to the page in which they are used, as such there is no one rule fits all. On implementation we found having 0 margin or the browser defaults on all the h tags to be not useful so we added a default of 1em 0.

Worth noting that we have spotted the issue of the grid importing much more than just the grid so we have swapped the import order of the docs for now.

@ariley90 @malcs-sky

### CODE REVIEW

**Comment by bentidy** [comment]
Is `.typography-example` needed?

**Comment by nowarkdev** [comment]
@bentidy Good spot, redundant now so we have just removed it.

**Comment by peter-mouland** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nowtv/nowtv-styleguide",
-  "version": "2.1.3",
+  "version": "2.1.4",
```
Comment (peter-mouland) [inline package.json]: this should be outside of the PR - as you don't know when the PR will get merged.

**Comment by bentidy** [comment]
:+1:

**Pull request merged**